Framework/PythonInterface/mantid/plots/helperfunctions.py

```diff
@@ -325,10 +325,13 @@ def get_md_data2d_bin_bounds(workspace, normalization, indices=None):
     """
     coordinate, data, _ = get_md_data(workspace, normalization, indices, withError=False)
     assert len(coordinate) == 2, 'The workspace is not 2D'
-    return coordinate[0], coordinate[1], data
+    if transpose:
```

Comment:
Is `transpose` better done with the decorator pattern that replaces changes to three different functions?